recipes/itk/all/patches/0001-find-conan-packages.patch

```diff
@@ -0,0 +1,157 @@
+--- Modules/ThirdParty/Eigen3/itk-module-init.cmake
++++ Modules/ThirdParty/Eigen3/itk-module-init.cmake
+@@ -1,3 +1,3 @@
+ if(ITK_USE_SYSTEM_EIGEN)
+-  find_package(Eigen3 REQUIRED CONFIG)
++  find_package(Eigen3 REQUIRED)
+ endif()
+--- Modules/ThirdParty/DCMTK/itk-module-init.cmake
++++ Modules/ThirdParty/DCMTK/itk-module-init.cmake
+@@ -24,7 +24,7 @@
+ if(ITK_USE_SYSTEM_DCMTK)
+   # Use local FindDCMTK.cmake.
+   list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_LIST_DIR}/CMake")
+-  find_package(DCMTK REQUIRED NO_MODULE)
++  find_package(DCMTK REQUIRED)
+ else(ITK_USE_SYSTEM_DCMTK)
+   # Copied and mofified from DCMTK/CMake/3rdparty.cmake
+   if(NOT DEFINED DCMTK_USE_ICU)
+--- Modules/ThirdParty/Eigen3/CMakeLists.txt
++++ Modules/ThirdParty/Eigen3/CMakeLists.txt
+@@ -5,7 +5,7 @@
+ mark_as_advanced(ITK_USE_SYSTEM_EIGEN)
+ 
+ if(ITK_USE_SYSTEM_EIGEN)
+-  set(_eigen_itk_target Eigen3::Eigen)
++  set(_eigen_itk_target Eigen3::Eigen3)
```

Comment:
`eigen` recipe has been fixed, you can rely on `cmake_find_package` and `cmake_find_package_multi` to properly define `Eigen3::Eigen` imported target.

Reply:
Thanks for fixing this.